app/src/main/java/com/mensinator/app/DatabaseUtils.kt

```diff
@@ -180,7 +181,15 @@ object DatabaseUtils {
             VALUES
             ('screen_protection', 'Protect screen', '1', '3', 'SW')
         """)
-
-
+    }
+    fun databaseVersion9(db: SQLiteDatabase) {
+        //Insert new row for custom period notification message
+        db.execSQL(
+            """
+            INSERT INTO app_settings(setting_key, setting_label, setting_value, group_label_id, setting_type)
+            VALUES
+            ('period_notification_message', 'Period Notification Message', "", '2', 'TX')
```

Comment:
Since we're going to be saving a "key" as a setting_value I would suggest something without spaces.

Reply:
Changed the setting_value, let me know if it's still not good

Reply:
@EmmaTellblom Please check this